Fix Using CSS gradients and improve grammar (#16277)
mdn/content: files/en-us/web/css/css_images/using_css_gradients/index.md

#### Example: closest-side for circles

> This example uses `closest-side`, which makes the circle's size to be the distance between the starting point (the center) and the closest side. The circle's radius is the distance between the center of the gradient and the closest edge, which due to the positioning of the 25% from the left and 25% from the bottom, is closest to the bottom, since the height in this case is narrower than the width.

Reviewer: Good catch!
I think we need to simplify the paragraph because the second sentence is mixing too many clauses. Also, it is repeating the fact "distance between center and closest edge" in both the sentences.
```suggestion
This example uses `closest-side`, which makes the circle's radius to be the distance between the center of the gradient and the closest side. In this case the radius is the distance between the center and the bottom edge, because the gradient is placed 25% from the left and 25% from the bottom, and the div element's height is smaller than the width.
```
